feedhenry/fh-mbaas-api
# task(Upgrad fh-db version): RHMAP-20779 - Upgrade from 3.3.0 to 3.3.1

# Jira link(s)
https://issues.jboss.org/browse/RHMAP-20779

# What
* Upgrad fh-db version

# Why
* Use the latest version to get bug and CEV fix

# How
Upgrade from `3.3.0` to `3.3.1`

# Verification Steps
* Check if the CI process will run the unit tests will finish with success. 
* No break changes
